src/components/common/withAuthentication/index.jsx

```diff
@@ -91,20 +89,7 @@ withAuthentication.propTypes = {
 
 const withSaml = WrappedComponent => (
   props => (
-    <StaticQuery
-      query={graphql`
-        query {
-          site {
-            siteMetadata {
-              providerSlug
-            }
-          }
-        }
-      `}
-      render={data => (
-        <WrappedComponent providerSlug={data.site.siteMetadata.providerSlug} {...props} />
-      )}
-    />
+    <WrappedComponent providerSlug={process.env.IDP_SLUG} {...props} />
```

Comment:
I'm wondering if it'd make sense to handle the case when `process.env.IDP_SLUG` is not set, perhaps just `console.error`'ing that you need to set the IDP_SLUG? Otherwise, will the wrapped component(s) be able to handle an empty `providerSlug` prop?